examples/examples/z_sub_liveliness.rs

```diff
@@ -39,9 +36,7 @@ async fn main() {
         .await
         .unwrap();
 
-    println!("Enter 'q' to quit...");
-    let mut stdin = async_std::io::stdin();
-    let mut input = [0_u8];
+    println!("Press CTRL-C to quit...");
     loop {
         select!(
```

Comment:
Select is no longer necessary, just have a `while let Ok(sample) = subscriber.recv().await { .. }`.